### PR TITLE
landing page form input

### PR DESCRIPTION
**Pull request by schittlur**

### CODE REVIEW

**Review by rglasgow-deloitte — APPROVED** [review]

File: src/components/CourseInformation/CourseInformation.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        name: item.display_name,
+        value: getCourseDetailValue(item.field_name, courseData) || "",
+      };
+    });
+  }
+  return (
+    <div className="content-section">
+      <div className="row content-panel course-detail">
+        <div className="inner-content">
+          <h4>{courseData.Course.CourseTitle}</h4>
```
Comment (jdiouf) [inline src/components/CourseInformation/CourseInformation.js:104]: Just to clarify, we're not using the configured course detail fields yet?

File: src/components/CourseInformation/RelatedCourses/RelatedCourses.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import ExperienceCard from "../../ExperienceCard/ExperienceCard";
+
+const RelatedCourses = (props) => {
+  // Expects an array of courses
+  const courses = props.data;
+
+
+  // If the data is not there 
+  if (!courses) {
+    return <div>Loading...</div>;
```
Comment (jdiouf) [inline src/components/CourseInformation/RelatedCourses/RelatedCourses.js:10]: The loading should take place in the parent component/wherever the API call is being made. If it's loading or errored out, don't render this component at all, else render this component.

File: src/components/CourseInformation/RelatedCourses/RelatedCourses.test.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+describe("RelatedCourses", () => {
```
Comment (jdiouf) [inline src/components/CourseInformation/RelatedCourses/RelatedCourses.test.js:21]: Once we make the change above, we can remove this test for a loading state

File: src/components/CourseInformation/CourseInformation.js (hunk 1 above)
Comment (rglasgow-deloitte) [reply src/components/CourseInformation/CourseInformation.js:104]: That is in my branch. I did not want to impede the ability of testing

File: src/components/CourseInformation/RelatedCourses/RelatedCourses.js (hunk 2 above)
Comment (rglasgow-deloitte) [reply src/components/CourseInformation/RelatedCourses/RelatedCourses.js:10]: I agree. We will remove/refactor that

**Review by jdiouf — APPROVED** [review]

**Pull request merged**